### PR TITLE
CASMTRIAGE-4052 Docs PR

### PR DESCRIPTION
**Pull request by cdelatte-hpe**
# Description
This has the blurb about in-family ceph upgrades and the process invovled.

# Checklist Before Merging

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [ x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Review by leliasen-hpe — APPROVED** [review]

**Review by rgross727 — APPROVED** [review]

File: upgrade/Stage_1.md [hunk 1]
```diff
@@ -40,6 +40,18 @@ Note that the progress for the current stage will not show up in Argo before the
...
+
+## In-family Upgrade requirement
+
+***This is only required for in-family CSM upgrades.  This typically pertains to testers, early adopters, and internal testing.***
+
+**IMPORTANT:** If the upgrade is an in-family CSM upgrade, then you will need to run the following to repoint the Ceph cluster to use the Ceph container image stored in Nexus.  The issue stems from slightly different `sha` values for the Ceph containers for in-family CSM storage node images which will prevent the Ceph containers from starting. This will utilize the upgrade procedure to the same version of Ceph as it has built in checks and health monitoring to better manage this rolling repoint of the Ceph containers.  
+
+(ncn-s001#)
+```bash
+   /srv/cray/scripts/common/cubs_tool.py --version v16.2.9 --registry registry.local/artifactory.algol60.net/csm-docker/stable/quay.io/ --upgrade  --in_family_override
```
Comment (rustydb) [inline upgrade/Stage_1.md:52]: Remove leading whitespace.

File: upgrade/Stage_1.md [hunk 2]
```diff
@@ -40,6 +40,18 @@ Note that the progress for the current stage will not show up in Argo before the
...
 For more information, see [Using the Argo UI](../operations/argo/Using_the_Argo_UI.md) and [Using Argo Workflows](../operations/argo/Using_Argo_Workflows.md).
 
+
+## In-family Upgrade requirement
+
+***This is only required for in-family CSM upgrades.  This typically pertains to testers, early adopters, and internal testing.***
+
+**IMPORTANT:** If the upgrade is an in-family CSM upgrade, then you will need to run the following to repoint the Ceph cluster to use the Ceph container image stored in Nexus.  The issue stems from slightly different `sha` values for the Ceph containers for in-family CSM storage node images which will prevent the Ceph containers from starting. This will utilize the upgrade procedure to the same version of Ceph as it has built in checks and health monitoring to better manage this rolling repoint of the Ceph containers.  
+
+(ncn-s001#)
```
Comment (rustydb) [inline upgrade/Stage_1.md:50]: The backticks need an empty newline above them. Maybe just move this to:

```markdown
Run this on `ncn-s001`

\`\`\`bash
/srv/cray/scripts/common/cubs_tool.py --version v16.2.9 --registry registry.local/artifactory.algol60.net/csm-docker/stable/quay.io/ --upgrade  --in_family_override
\`\`\`
```

**Review by rustydb — CHANGES_REQUESTED** [review]

File: upgrade/Stage_1.md [hunk 3]
```diff
@@ -40,6 +40,18 @@ Note that the progress for the current stage will not show up in Argo before the
 
 For more information, see [Using the Argo UI](../operations/argo/Using_the_Argo_UI.md) and [Using Argo Workflows](../operations/argo/Using_Argo_Workflows.md).
 
+
+## In-family Upgrade requirement
```
Comment (denniswalker) [inline upgrade/Stage_1.md:44]: Update "in-family" to say a more recent version of the same release of CSM, e.g. CSM V1.3.0-rc.2 to CSM V1.3.0-rc.3

Comment (cdelatte-hpe) [reply upgrade/Stage_1.md:44]: Please check if the wording is good enough  @denniswalker

**Comment by rustydb** [comment]
The commits were confusing the backport function, I just squashed them which had a lot of conflicts weirdly enough.

**Pull request merged**